18-hlt-intro.md

```diff
@@ -60,54 +69,56 @@ report = reports.decReport('Hlt1TrackAllL0Decision')
 print report.decision()
 ~~~
 
-The HLT1 selection that was most efficient for hadronic charm and beauty decays
-in Run-I was called Hlt1TrackAllL0. Use the advance function to find an event
-that was accepted by that trigger selection.
+The HLT1 selections that are most efficient for hadronic charm and beauty decays
+in Run-II are called Hlt1TrackMVA and Hlt1TwoTrackMVA. Use the advance function
+to find an event that was accepted by either of these trigger selections.
 
 The DecReports only contain the decisions, the candidates are stored in the
 SelReports ("Hlt{1,2}/SelReports"). Get the Hlt1 SelReports from the event store
-and retrieve the one for Hlt1TrackAllL0 using the selReport function,
+and retrieve the one for one of the TrackMVA selections using the selReport function,
 analogously to how the DecReport was retrieved above.
 
 The SelReports store candidates in a tree of sub-structures, which can be
 accessed using the substructure member funtion of a report. Any SelReport has at
 least one level of sub-structure. The sub-structure is internally stored in
 SmartRefs, which can be derefereced using their "data" method. Let's have a look
-at a SelReport for Hlt1TrackAllL0.
+at a SelReport for an TrackMVA selection.
```

Comment:
We haven't met `SmartRef` objects yet. This might warrant a call-out box. You can make these with:

```
> ## SmartRefs? {.callout}
> Text inside a call-out.
> ```
> can have code as well
> ```
```

Also “ derefereced” → “dereferenced”.

Also “an TrackMVA” → “a TrackMVA”.